packages/orbit-components/src/InputField/index.tsx

```diff
@@ -143,154 +143,162 @@ const InputField = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
-            type === TYPE_OPTIONS.PASSPORTID && "tabular-nums tracking-[2px]",
+            "orbit-input-field-input-container",
+            "flex flex-row items-center justify-between",
+            "relative box-border",
+            "h-form-box-normal text-form-element-normal",
+            disabled ? "cursor-not-allowed" : "cursor-text",
+            disabled || readOnly
+              ? "text-form-element-disabled-foreground"
+              : "text-form-element-filled-foreground",
           )}
-          data-test={dataTest}
-          required={required}
-          data-state={
-            insideInputGroup && typeof error === "undefined"
-              ? undefined
-              : getFieldDataState(!!error)
-          }
-          onChange={onChange}
-          onFocus={handleFocus}
-          onBlur={onBlur}
-          onKeyUp={onKeyUp}
-          onKeyDown={onKeyDown}
-          onSelect={onSelect}
-          onMouseUp={onMouseUp}
-          onMouseDown={onMouseDown}
-          name={name}
-          type={getDOMType(type)}
-          value={value}
-          defaultValue={defaultValue}
-          placeholder={placeholder}
-          disabled={disabled}
-          min={minValue}
-          max={maxValue}
-          minLength={minLength}
-          maxLength={maxLength}
-          ref={ref}
-          tabIndex={tabIndex !== undefined ? Number(tabIndex) : undefined}
-          list={list}
-          aria-label={ariaLabel}
-          aria-describedby={shown ? `${inputId}-feedback` : undefined}
-          aria-invalid={error ? true : undefined}
-          aria-autocomplete={ariaAutocomplete}
-          aria-haspopup={ariaHasPopup}
-          aria-expanded={ariaExpanded}
-          aria-controls={ariaControls}
-          aria-activedescendant={ariaActiveDescendant}
-          readOnly={readOnly}
-          autoCapitalize="off"
-          autoCorrect="off"
-          role={role}
-          autoComplete={autoComplete}
-          // eslint-disable-next-line jsx-a11y/no-autofocus
-          autoFocus={autoFocus}
-          id={inputId}
-          inputMode={inputMode}
-          {...dataAttrs}
-        />
-        {suffix && <Suffix disabled={disabled}>{suffix}</Suffix>}
-        <FakeInput error={error} disabled={disabled} />
+          ref={label ? null : labelRef}
+        >
+          {inlineLabel && !tags && (error || help) ? (
+            <Prefix>
+              {help && !error && (
+                <div className="flex" ref={iconRef}>
+                  <InformationCircle color="info" size="small" />
+                </div>
+              )}
+              {error && (
+                <div className="flex" ref={iconRef}>
+                  <AlertCircle color="critical" size="small" />
+                </div>
+              )}
+            </Prefix>
+          ) : (
+            prefix && <Prefix>{prefix}</Prefix>
+          )}
+          {label && inlineLabel && (
+            <label
```

Comment:
Something is off about this implementation. When one presses the label, the input should receive focus. And if it has an help or error tooltip, it should be triggered. But with this change this is not happening 😕 Please take a look at it 🙏

Reply:
Should be fixed. The tricky part was, that the component includes FakeInput  (it's responsible for focus outline) with absolute positioning and it covered the label (there was not set z-index), so click was not possible.